Odata-docs/webapi-8/tutorials/using-utf8jsonwriter-to-improve-serialization-performance.md

```diff
@@ -0,0 +1,33 @@
+---
+title:  "Using Utf8JsonWriter to improve serialization performance"
+description: Learn how to configure ASP.NET Core OData 8 to use Utf8JsonWriter to improve serialization performance. 
+date:   2023-08-11
+ms.date: 08/11/2023
+author: habbes
+ms.author: clhabins
+---
+
+# Using Utf8JsonWriter to improve serialization performance
+
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+`Microsoft.OData.Core` version 7.12.2 introduced a new JSON writer that writes directly to a `Stream` rather than using a `TextWriter`. It also introduced a default implementation of this "stream-based" JSON writer that’s based on .NET’s [`Utf8JsonWriter`](/dotnet/api/system.text.json.utf8jsonwriter). This writer offers better performance than the default JSON writer used in `Microsoft.OData.Core`. To learn more, [visit this page](../../odatalib/v7/using-utf8jsonwriter-for-better-performance.md). In this article, we are going to show you how to configure your ASP.NET Core OData application to use the `Utf8JsonWriter`.
```

Comment:
The first sentence... I'd rather you talked about the new JSON writer being based on Utf8JsonWriter (or being a wrapper for Utf8JsonWriter) than talk about it writing directly to a `Stream` as opposed to using a `TextWriter`.

Reply:
Makes sense. Should I mention the `Stream` and `TextWriter` differences at all? I did so to help explain the name. But I could also remove that and just mention that the writer is based on `Utf8JsonWriter.